### PR TITLE
v1.5.2

### PR DESCRIPTION
**Pull request by grahamhency**
###### `v1.5.2`

# 🚢 New Features

### App Drawer
- Added the ability to generate an app drawer in the side nav #469 

# 🐞 Bug Fixes

### Select
- Fixed an issue where the select inputs were 4px taller than the other inputs #470 
- Fixed an issue where the select dropdown panel is 4px to the left when using `appendTo="body"` #452 

### Table
- Fixed an issue where the `GoTableSearchConfig` was not properly exposed in `public_api`. Additionally fixed a couple other small issues around the table and searching. #467

### CODE REVIEW

File: projects/go-style-guide/src/app/features/ui-kit/components/table-docs/components/table-searching/table-searching.component.html [hunk 1]
```diff
@@ -21,11 +21,13 @@ <h4 class="go-heading-4">searchable</h4>
...
       </div>
 
-      <div class="go-column go-column--100">
+      <div class="go-column go-column--100 go-column--no-padding">
         <h4 class="go-heading-4">searchTerm</h4>
         <p class="go-body-copy">
           This is the term that is being searched for. This is useful when implementing server side integration, as it is emitted
           as part of the <code class="code-block--inline">GoTableConfig</code> object and can be used for server side filtering.
+          <br/>
+          <caption>See <a [routerLink]="['../server-integration']">Server Integration</a> for more details.</caption>
```
Comment (jaredami) [inline projects/go-style-guide/src/app/features/ui-kit/components/table-docs/components/table-searching/table-searching.component.html]: Should this have `class="go-link"` on it?

File: projects/go-lib/src/lib/components/go-side-nav/go-side-nav/go-side-nav.component.scss [hunk 2]
```diff
@@ -10,8 +10,7 @@
   flex-grow: 1;
   font-family: $secondary-font-stack;
   height: 100%;
-  overflow-y: auto;
-  padding: 2rem 0;
+  padding: 2rem 0 0 0;
```
[the remark this answers is not in the document]
Comment (grahamhency) [reply projects/go-lib/src/lib/components/go-side-nav/go-side-nav/go-side-nav.component.scss]: It should not, actually. We only need `padding-top`, so maybe we can update it to that.

File: projects/go-style-guide/src/app/features/ui-kit/components/layout-docs/components/layout-app-drawer/layout-app-drawer.component.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    <ng-container go-card-header>
+      <h2 class="go-heading-2">Example</h2>
+    </ng-container>
+    <div class="go-container" go-card-content>
+
+      <div class="go-column go-column--100 go-column--no-padding">
+        <p class="go-body-copy">
+          Working example of the below code can be found on the bottom left of the side navigation. For details on required
+          imports and adding the <code class="code-block--inline">&lt;go-side-nav&gt;</code> component please reference the
+          <a [routerLink]="['/ui-kit/layout/side-nav']">Side Nav Documentation</a>.
```
[the remark this answers is not in the document]
Comment (grahamhency) [reply projects/go-style-guide/src/app/features/ui-kit/components/layout-docs/components/layout-app-drawer/layout-app-drawer.component.html]: Created a PR for this.

File: projects/go-style-guide/src/app/features/ui-kit/components/table-docs/components/table-searching/table-searching.component.html (hunk 1 above)
Comment (grahamhency) [reply projects/go-style-guide/src/app/features/ui-kit/components/table-docs/components/table-searching/table-searching.component.html]: Created a PR for this.

File: projects/go-style-guide/src/app/features/ui-kit/components/layout-docs/components/layout-nav/layout-nav.component.html [hunk 4]
```diff
@@ -31,6 +31,14 @@ <h4 class="go-heading-4">menuItems</h4>
         <code [highlight]="bindings_menuItems"></code>
       </div>
 
+      <div class="go-column go-column--100">
+        <h4 class="go-heading-4">navAppDrawer</h4>
+        <p class="go-body-copy">
+          Optional binding to render app drawer within side navigation. Please reference
+          <a [routerLink]="['/ui-kit/layout/app-drawer']">App Drawer Documentation</a> for implementation details.
```
[the remark this answers is not in the document]
Comment (grahamhency) [reply projects/go-style-guide/src/app/features/ui-kit/components/layout-docs/components/layout-nav/layout-nav.component.html]: Created a PR for this.

File: projects/go-style-guide/src/app/features/ui-kit/components/layout-docs/components/layout-app-drawer/layout-app-drawer.component.html [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+      <h2 class="go-heading-2">Usage</h2>
+    </ng-container>
+    <div class="go-container" go-card-content>
+
+      <div class="go-column go-column--100">
+        <p class="go-body-copy go-body-copy--no-margin">
+          This page details how to use the <code class="code-block--inline">navAppDrawer</code> binding associated with
+          the <code class="code-block--inline">&lt;go-side-nav&gt;</code> component. For any other information related to
+          <code class="code-block--inline">&lt;go-side-nav&gt;</code>, please reference the
+          <a [routerLink]="['/ui-kit/layout/side-nav']">Side Nav Documentation</a>.
```
[the remark this answers is not in the document]
Comment (grahamhency) [reply projects/go-style-guide/src/app/features/ui-kit/components/layout-docs/components/layout-app-drawer/layout-app-drawer.component.html]: Created a PR for this.

**Comment by grahamhency** [comment]
@jaredami I created a PR to merge into this with your suggested changes. See #474

**Pull request merged**